layouts/partials/posts/featured.html

```diff
@@ -7,7 +7,11 @@
         <span class="date">{{ .Date.Format (.Site.Params.DateFormat | default "January 2, 2006") }}</span>
         {{ end }}
         <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
+        {{ if .Description }}
         <p>{{ .Description }}</p>
+        {{ else }}
+        <p>{{ .Summary }}</p>
```

Comment:
I can't see a use of `Summary` in the example site. How does this differ from `Description`?

Or is this in case the existing markdown files are using `Summary` instead of `Description`?

Reply:
See point 1 in my answer below.